remoteworkitem/scheduler.go

```diff
@@ -46,26 +44,25 @@ func batchID() string {
 // ScheduleAllQueries fetch and import of remote tracker items
 func (s *Scheduler) ScheduleAllQueries() {
 	cr.Stop()
-	ts := models.NewGormTransactionSupport(s.db)
 
 	trackerQueries := fetchTrackerQueries(s.db)
 	for _, tq := range trackerQueries {
 		cr.AddFunc(tq.Schedule, func() {
 			tr := lookupProvider(tq)
 			for i := range tr.Fetch() {
-				transaction.Do(ts, func() error {
-
+				tx := s.db.Begin()
```

Comment:
Totally bogus! I guess I forgot this one when refactoring. Tests still pass, though, I guess transaction timeout comes later.